VMBackup/main/freezesnapshotter.py

```diff
@@ -114,6 +114,8 @@ def doFreezeSnapshot(self):
         all_failed = False
         unable_to_sleep = False
 
+        http_util = HttpUtil(self.logger)
```

Comment:
This is just to ensure HttpUtil singleton object is created before freeze.

Reply:
add the same as comment here

Reply:
Added